api/src/pcapi/core/users/commands.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from pcapi import settings
+import pcapi.core.fraud.api as fraud_api
+from pcapi.core.users import api
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏
```suggestion
import pcapi.core.users.api as users_api